CONTRIBUTING.md

```diff
@@ -1,8 +1,49 @@
+# Contributing
+
+Thanks for contributing to the Cloud Four Pattern Library!
+
+## Getting Started
+
+1. `npm ci` — to install dependencies
+1. `npm run storybook` — to run a local instance of Storybook
+
+## Building
+
+- `npm run build` — Builds CSS+JS for npm package
+- `npm run build-storybook` — Creates a static storybook site build, for example for publishing the pattern library to Netlify
+
+## Project Structure
+
+```
+cloudfour.com-patterns
+├── .github               # GitHub workflows and templates
+├── .storybook
+│   ├── main.js           # Settings for Storybook UI
+│   └── preview.js        # Settings for story previews
+├── .theo                 # Settings for design token processing
+├── gulpfile.js
+│   └── tasks/*.js        # Complex build tasks
+├── src
+│   ├── **/*.scss         # Styles (Sass)
+│   ├── **/*.stories.mdx  # Documentation (Storybook Docs)
+│   ├── **/*.twig         # Templates (Twig)
+│   └── **/*.yml          # Design tokens (Theo)
+├── .editorconfig         # Low-level code consistency
+├── .nvmrc                # Node version (used by Netlify)
+├── .svgo.yml             # Inline SVG optimization settings
+├── README.md             # ⬅️ You are here!
+├── netlify.toml          # Netlify build settings
+└── package.json          # Project info and dependencies
+```
+
 ## Publishing to npm
 
 1. `git checkout v-next`
 2. `git pull`
 3. Make sure you have a clean working tree (`git status` should show no changes)
-4. `npm publish` - This will automatically install and compile everything, run linting, and publish
+4. `npm version` - This will bump the version number in `package.json` and `package-lock.json`
```

Comment:
Does this instruction need to specify anything about semver and the options for bumping versions based on types of changes?

Reply:
I don't think so — explaining how npm works is probably outside the scope of a readme on how to install our package.